tests/director/tests/formation_constraint_api_test.go

```diff
@@ -287,9 +287,27 @@ func TestListFormationConstraintsForFormationTemplate(t *testing.T) {
 	defer fixtures.CleanupFormationConstraint(t, ctx, certSecuredGraphQLClient, constraint.ID)
 	require.NotEmpty(t, constraint.ID)
 
+	// Assert no constraints attached
+	t.Logf("Get formation template with name %q and id %q, and assert there are no constraints attached to it", formationTemplate.Name, formationTemplate.ID)
+	formationTemplateOutput := fixtures.QueryFormationTemplateWithConstraints(t, ctx, certSecuredGraphQLClient, formationTemplate.ID)
```

Comment:
We have to regenerate the examples as the query has changed. Running the scrip should be sufficient as the graphqlizer is already adapted